integration-tests/src/test/common.ts

```diff
@@ -2,8 +2,9 @@ import { Wallet } from 'ethers';
 import { Guid } from 'js-guid';
 import { BN } from '@polkadot/util';
 import { KeyringPair } from '@polkadot/keyring/types';
-import { Blockchain, LoanTerms } from 'creditcoin-js/model';
+import { Blockchain, LoanTerms, DealOrderId } from 'creditcoin-js/model';
```

Comment:
I think prefixing with `../../../` to point to the actual file on disk will solve your issues. That's how some of the utils functions are imported in the existing test suite. They are not built as a JS library.